File: Programming Basics/Conditional Statements - Exercises/05.GodzillaVsKong.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        double budget = double.Parse(Console.ReadLine());
        int actors = int.Parse(Console.ReadLine());
        double pricePerActor = double.Parse(Console.ReadLine());

        double decorPrice = budget * 0.1;

        if (actors > 150)
        {
            pricePerActor = pricePerActor - (pricePerActor * 0.1);
        }

        double neededMoney = decorPrice + (pricePerActor * actors);

        if (budget >= neededMoney)
        {
            Console.WriteLine("Action!");
            Console.WriteLine($"Wingard starts filming with {budget - neededMoney:f2} leva left.");
        }
        else if (budget < neededMoney)
        {
            Console.WriteLine("Not enough money!");
            Console.WriteLine($"Wingard needs {neededMoney - budget:f2} leva more.");
        }
    }
}

File: Programming Basics - Python/Conditional Statements - Exercises/05.GodzillaVsKong.py
movie_budget = float(input())
extras_count = int(input())
extras_clothing_cost = float(input())

decoration_budget = movie_budget * 0.10

if extras_count >= 150:
    extras_clothing_cost -= extras_clothing_cost * 0.10

total_cost = extras_count * extras_clothing_cost + decoration_budget

if movie_budget < total_cost:
    print('Not enough money!')
    print(f'Wingard needs {total_cost - movie_budget:.2f} leva more.')

else:
    print('Action!')
    print(f'Wingard starts filming with {movie_budget - total_cost:.2f} leva left.')